kubespawner/spawner.py

```diff
@@ -909,6 +931,8 @@ def start(self):
         # FIXME: Have better / cleaner retry logic!
         retry_times = 4
         pod = yield self.get_pod_manifest()
+        if self.augment_pod:
+            pod = self.augment_pod(self, pod)
```

Comment:
Why also pass `self`?

Reply:
So the augmenter can access other traitlets, and also call methods to get things like auth_state...